packages/slice-machine/src/features/onboarding/OnboardingTutorial/OnboardingTutorial.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, Icon, Text } from "@prismicio/editor-ui";
+
+import { useOnboardingCardVisibilityExperiment } from "@/features/onboarding/useOnboardingCardVisibilityExperiment";
+import { useMarketingContent } from "@/hooks/useMarketingContent";
+
+export function OnboardingTutorial() {
+  const { variant } = useOnboardingCardVisibilityExperiment();
+  const { tutorial } = useMarketingContent();
+
+  const color = variant === "dark" ? "white" : "grey11";
+  const url = tutorial?.url;
+
+  if (url === undefined) return null;
+
+  return (
+    <Box justifyContent="center" padding={{ block: 6 }} gap={4}>
+      <Text href={url} variant="smallBold" color={color}>
+        Or watch our full course
+      </Text>
+      <Icon name="playCircle" color={color} size="small" />
+    </Box>
```

Comment:
💡 The Icon should also be clickable, so let try something like this. Kept the Block because of the padding.

```suggestion
    <Box justifyContent="center" padding={{ block: 6 }}>
      <Text href={url} variant="smallBold" color={color}>
        Or watch our full course{' '}
        <Icon name="playCircle" color={color} size="small" />
      </Text>
    </Box>
```

Reply:
good point!
updated